<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT_IDIClass</class>
 <widget class="QMainWindow" name="QT_IDIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>633</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>QT_IDI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_CameraDevices">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConnectedCamera">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DisConnectedCamera">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DisConnected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartGrab">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>StartGrab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StopGrab">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>StopGrab</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_SingleGrab">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Single</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_ContinueGrab">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>441</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QT_IDI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_ConnectedCamera</sender>
   <signal>clicked()</signal>
   <receiver>QT_IDIClass</receiver>
   <slot>ConnectedCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_DisConnectedCamera</sender>
   <signal>clicked()</signal>
   <receiver>QT_IDIClass</receiver>
   <slot>DisConnectedCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>868</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>810</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_StartGrab</sender>
   <signal>clicked()</signal>
   <receiver>QT_IDIClass</receiver>
   <slot>GrabOnce()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_StopGrab</sender>
   <signal>clicked()</signal>
   <receiver>QT_IDIClass</receiver>
   <slot>GrabContinue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ConnectedCamera()</slot>
  <slot>DisConnectedCamera()</slot>
  <slot>GrabOnce()</slot>
  <slot>GrabContinue()</slot>
 </slots>
</ui>
